scripts/src/java/org/oppia/android/scripts/common/BUILD.bazel

```diff
@@ -36,6 +36,10 @@ kt_jvm_library(
         "CommandResult.kt",
     ],
     visibility = ["//scripts:oppia_script_library_visibility"],
+    deps = [
+        ":script_background_coroutine_dispatcher",
+        "//third_party:org_jetbrains_kotlinx_kotlinx-coroutines-core",
```

Comment:
Not technically, but we actually want to move toward explicit (rather than implicit from the transitive closure) dependencies since it leads to a more peformant and predictable build graph. In the Bazel world this is usually called 'strict dependencies' and we'll be enabling it later in the Bazel chain. I'm going to refrain from removing this since it'll just be readded in a later PR, anyway.

Thanks for pointing it out!